pyav-example/run_pyav.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+# Function to detect Linux distribution
+detect_distro() {
+    if [ -f /etc/os-release ]; then
+        . /etc/os-release
+        echo $ID
+    elif [ -f /etc/redhat-release ]; then
+        echo "rhel"
+    elif [ -f /etc/debian_version ]; then
+        echo "debian"
+    else
+        echo "unknown"
+    fi
+}
+
+# Install system dependencies based on distribution
+DISTRO=$(detect_distro)
+
+case $DISTRO in
+    "fedora"|"rhel"|"centos"|"rocky"|"almalinux")
+        if command -v dnf >/dev/null 2>&1; then
+            dnf install -y python3.12 python3.12-devel python3.12-pip gcc-toolset-13 libjpeg-devel freetype-devel
+            source /opt/rh/gcc-toolset-13/enable
+        else
+            yum install -y python3.12 python3.12-devel python3.12-pip gcc-toolset-13 libjpeg-devel
+            source /opt/rh/gcc-toolset-13/enable
+        fi
+        ;;
+    "ubuntu"|"debian")
+        apt update
+        apt install -y python3.12 python3.12-dev python3-pip python3.12-venv gcc libjpeg-dev libgfortran5 g++
+        ;;
```

Comment:
Let's consider using `sudo` for installing system libraries, instead of assuming `root` user running this script.